Formula/lib/libscrypt.rb

```diff
@@ -17,16 +17,25 @@ class Libscrypt < Formula
   end
 
   def install
+    # `-Os` leads to bugs. https://github.com/technion/libscrypt/issues/60
+    # TODO: Add `ENV.O2` to `brew`.
+    ENV["HOMEBREW_OPTIMIZATION_LEVEL"] = "O2"
```

Comment:
We could do `ENV.O1`. I've tested and `-O1` does not trigger the bug.

Reply:
We should do `ENV.O1`. We don't want to add `ENV.O2` to Homebrew/brew because it's already the default on Linux.

Reply:
We should also not use `HOMEBREW_OPTIMIZATION_LEVEL` like this in formulae.